pyrefinebio/hlf.py

```diff
@@ -0,0 +1,88 @@
+import pyrefinebio
+import re
+
+from pyrefinebio import Dataset, Compendia
+from pyrefinebio.http import download_file
+from pyrefinebio.exceptions import DownloadError
+
+def help(entity=None):
+    """Help
+
+    Prints out information about pyrefinebio's classes and functions.
+    To get information about a class method, pass in the class name and method name  
+    separated by either a space or a `.`
+    
+    usage:
+
+        getting info about classes:
+
+        >>> pyrefinebio.help("Sample")
+        
+        gettting info about class methods:
+
+        >>> pyrefinebio.help("Sample.get")
+        or
+        >>> pyrefinebio.help("Sample get")
```

Comment:
This works for functions as well as methods right? Could we include an example of that:  `pyrefinebio.help("download_dataset")`